delira/training/base_trainer.py

```diff
@@ -310,18 +370,31 @@ def _train_single_epoch(self, batchgen: Augmenter, epoch,
             iterable = enumerate(batchgen)
 
         for batch_nr, batch in iterable:
```

Comment:
I would prefer consistent naming: either `batch_nr` or `iter_num` (so closure and callback have consistent names for the same thing)

Reply:
I prefer `iter_num`